package.json

```diff
@@ -202,7 +203,7 @@
     "react-native-sqlite-2": "^3.5.2",
     "react-native-svg": "^9.13.6",
     "react-native-svg-transformer": "^0.14.3",
-    "react-native-swipeable-lists-gd": "0.0.3",
+    "react-native-swipeable-list": "^0.1.2",
```

Comment:
We used own customized version of think package
Check why please. Didn’t you broke something reverting to the “stock” version

Reply:
@roma-hladilka ^^^

Reply:
added a patch